00-Introducing-HOOMD-blue/05-Compressing-the-System.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p>No, we don't. Fluid representations should be avoided when teaching novice users.</p>

Reply:
Okay, I was thinking that this would be the standard way of creating triggers, but I guess we do want to encourage the understanding of what a trigger is. Thanks.

Reply:
The trouble with showing the first example a user seeds is `trigger=10`, then they will just assume that trigger is always a periodic trigger with a period given like that.